lua/lspconfig/configs/julials.lua

```diff
@@ -1,19 +1,19 @@
-local util = require 'lspconfig.util'
+local util = require('lspconfig.util')
 
 local root_files = { 'Project.toml', 'JuliaProject.toml' }
 
 local function activate_env(path)
-  assert(vim.fn.has 'nvim-0.10' == 1, 'requires Nvim 0.10 or newer')
+  assert(vim.fn.has('nvim-0.10') == 1, 'requires Nvim 0.10 or newer')
   local bufnr = vim.api.nvim_get_current_buf()
-  local julials_clients = vim.lsp.get_clients { bufnr = bufnr, name = 'julials' }
+  local julials_clients = vim.lsp.get_clients({ bufnr = bufnr, name = 'julials' })
   assert(
     #julials_clients > 0,
     'method julia/activateenvironment is not supported by any servers active on the current buffer'
   )
   local function _activate_env(environment)
     if environment then
       for _, julials_client in ipairs(julials_clients) do
-        julials_client.notify('julia/activateenvironment', { envPath = environment })
+        util.client_proxy(julials_client).notify('julia/activateenvironment', { envPath = environment })
```

Comment:
Definitely not in favor of this PR if it requires `client_proxy` in the configs themselves. That just introduces even more migration later.

If there's a way to make it transparent to configs, that's fine. Otherwise we should just wait until we can bump the minimum required Nvim to 0.11

Reply:
Yes. The current command design is too fragile. The client is necessary here. So it is better to provide a default wrapper function with the `client` and `bufnr` as the parameter so that we can easily handle any upstream changes on the client object internally.  

```lua
      switch_source_header(client, bufnr),   
````